src/Exceptions/InvalidConfiguration.php

```diff
@@ -3,12 +3,13 @@
 namespace Spatie\Activitylog\Exceptions;
 
 use Exception;
-use Spatie\Activitylog\Models\Activity;
+use Illuminate\Database\Eloquent\Model;
+use Spatie\Activitylog\Contracts\Activity;
 
 class InvalidConfiguration extends Exception
 {
     public static function modelIsNotValid(string $className)
     {
-        return new static("The given model class `$className` does not extend `".Activity::class.'`');
+        return new static("The given model class `$className` does not implements `".Activity::class.'` and extend `'.Model::class.'`');
```

Comment:
If we're doing proper `instanceof` checks in the service provider, I don't think this exception is necessary, maybe I'm missing something? (@freekmurze ?)

Reply:
But this exception is thrown by the service-provider!? So what-else do you want to do if the model isn't right?

Reply:
This exception was thrown by the server provider also in the original package. I had only changed the message to fit the new requirements (implements `Spatie\Activitylog\Contracts\Activity` Interface and extend `Spatie\Activitylog\Contracts\Activity`)

Reply:
Looks like I missed that, maybe I should drink my morning coffee before going through code 😅

Reply:
Ok I could change this into my code and push the commits next week.

Reply:
Nitpick on the language used. `implements` should be `implement`.  Replace `and extend` with `or it does not extend`.

Reply:
Sorry for my bad english :( 
I'll correct this.
Thanks